### PR TITLE
feat(example): add gmp example scripts

### PR DESCRIPTION
**Pull request by ahramy**
[AXE-6535](https://axelarnetwork.atlassian.net/browse/AXE-6535)

[AXE-6535]: https://axelarnetwork.atlassian.net/browse/AXE-6535?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: stellar/README.md [hunk 1]
```diff
@@ -173,6 +180,27 @@ node stellar/its.js set-trusted-address [chain-name] [trusted-address]
...
 ```bash
 node stellar/its.js remove-trusted-address [chain-name]
 ```
+
+### Example
+
+#### GMP - Send
+
+```bash
+node stellar/gmp.js send [destination-chain] [destination-address] [payload] [gas-token-address] [gas-fee-amount]
```
Comment (ahramy) [inline stellar/README.md:193]: # approve should be done before executing the gmp call
soroban contract invoke --network testnet --id CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC --source-account wallet -- approve --from $CALLER_ADDRESS --expiration_ledger 3110400 --spender $GAS_SERVICE --amount 100

File: axelar-chains-config/info/devnet-amplifier.json [hunk 2]
```diff
@@ -620,6 +620,14 @@
         "initializeArgs": {
           "gasCollector": "GALITAYSFTLEVXIBTF5FVUIV7PFVF7VF5VSZVFYCXA7PXSZ2LZMY54VG"
         }
+      },
+      "example": {
+        "address": "CDV63R2YDFEYWXM5S4C3MLW65SIOKZYAY67NUGJ4QOGJXK6LJOSZAWLJ",
```
Comment (milapsheth) [inline axelar-chains-config/info/devnet-amplifier.json:625]: old or new deployment?

**Review by milapsheth — APPROVED** [review]

File: stellar/README.md [hunk 3]
```diff
@@ -173,6 +180,27 @@ node stellar/its.js set-trusted-address [chain-name] [trusted-address]
...
+
+# Example
+node stellar/gmp.js send avalanche 0xba76c6980428A0b10CFC5d8ccb61949677A61233 0x1234 CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC 1
+```
+
+#### GMP - Execute
+
```
[the remark this answers is not in the document]
Comment (ahramy) [reply stellar/README.md:227]: done. I think the best way is to separate out gmp and its instructions to it's own doc folder like SUI. will work on it later.

File: axelar-chains-config/info/devnet-amplifier.json (hunk 2 above)
Comment (ahramy) [reply axelar-chains-config/info/devnet-amplifier.json:625]: this is a new deployment with the latest code.

**Pull request merged**